src/main/java/browserview/BrowserComponent.java

```diff
@@ -487,9 +487,9 @@ public String getCurrentUrl() {
         return driver.getCurrentUrl();
     }
 
-    public void switchToConversationTab() {
+    public void switchToTab(String tabName) {
```

Comment:
Could this string be an enum instead? We don't want callers of this method to have to know how GitHub's DOM elements are named. The enum can be in GitHubURL and have a toString method which generates the string.